### PR TITLE
Made Deck component to refactor Minicard code

### PR DESCRIPTION
**Pull request by iturnmyscreenon**

### CODE REVIEW

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by iturnmyscreenon** [comment]
As requested, i stored the propTypes shape in propTypes.js and replaced powerstats as strings

File: frontend/src/components/propTypes.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  powerstats: PropTypes.shape({
+    intelligence: PropTypes.string,
+    strength: PropTypes.string,
+    speed: PropTypes.string,
+    durability: PropTypes.string,
+    power: PropTypes.string,
+    combat: PropTypes.string,
+  }),
+  image: PropTypes.shape({ url: PropTypes.string }),
+});
```
Comment (rocambille) [inline frontend/src/components/propTypes.js:31]: try:

```js
const cardPropType = PropTypes.shape({
  id: PropTypes.number,
  name: PropTypes.string,
  powerstats: PropTypes.shape({
    intelligence: PropTypes.string,
    strength: PropTypes.string,
    speed: PropTypes.string,
    durability: PropTypes.string,
    power: PropTypes.string,
    combat: PropTypes.string,
  }),
  image: PropTypes.shape({ url: PropTypes.string }),
});

const cardArrayPropType = PropTypes.arrayOf(cardPropType);
```

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/propTypes.js (hunk 1 above)
Comment (iturnmyscreenon) [reply frontend/src/components/propTypes.js:31]: Done.

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by iturnmyscreenon** [comment]
the import and export were fixed

**Pull request merged**